Zhuohang li

Stellaaali opened the pull request:
plz review this survey:))

Stellaaali committed:
Update 3_postnatal_care.do

update "delivery at home" coding based on Recode7.txt

Stellaaali committed:
Update 8_child_illness.do

-add some new specific names of formal/informal provider

Stellaaali committed:
Update 10_child_mortality.do

delete: can't find a corresponding variable to this command

Stellaaali committed:
Update 11_child_other.do

create new syntax for ITN variable:
match the dataset "child"(adhere the logic of dofile ""DHS_Recode_VII

Stellaaali committed:
Update 16_woman_cancer.do

Stellaaali committed:
some comprehensive changes

"11.do"
- wrote syntax for ITN

"3.do"
- made correctness

"8.do"
-add elements in current commands, details in dofile notes

Stellaaali committed:
blank-no real changes

blank

Stellaaali committed:
some changes

some changes made (no relation to content, just solve technical problems

Stellaaali committed:
Update DHS_Recode_VII.do

robin-wang commented on 8_child_illness.do:
```diff
@@ -45,6 +47,24 @@ if ~inlist(name,"Philippines2017","Ethiopia2016","Haiti2016") {
         replace c_diarrhea_pro = . if pro_dia == . 
 }    	
 
+ ****why here add:1. varlist range changed(x→z) 2. new label*******
```
This section needs some discussion, you could set up a time to walk me through your thinking when creating this code snippet.

robin-wang commented on Quality_control.do:
```diff
@@ -105,7 +105,7 @@ keep if _n == 1
 replace value_my = value_my*100 if varname_my != "w_bmi_1549"
 
-merge 1:1 surveyid varname_my ispreferred using "${SOURCE}/external/DHS.dta"
+merge 1:1 surveyid varname_my ispreferred using "${SOURCE}DHS-Liberia2019\external\DHS.dta"
```
Generally, we don't want to modify a hard-coded file path in the way you suggested. That's because we all rely on the same set of programmes to operate across the team.
It is better to reflect the path change in the global macro. If this is hard to understand, let's speak about this concept.

robin-wang commented:
I left more comprehensive remarks in the issue for Liberia2019 
https://github.com/wengxyu1030/DHS-Recode-VII/issues/32
Some of these issues need to be covered in a brief chat. You need to walk me through your thought process : )

robin-wang commented on 8_child_illness.do:
```diff
@@ -26,14 +26,16 @@
                  local lab: variable label `var' 
         replace `var' = . if ///
-                 regexm("`lab'","(other|shop|pharmacy|market|kiosk|relative|friend|church|drug|addo|hilot|traditional|cs private medical|cs public sector|no treatment)") ///
+                 regexm("`lab'","(other|shop|pharmacy|market|kiosk|relative|friend|church|drug|addo|hilot|traditional|cs private medical|cs public sector|private mobile clinic|no treatment)") ///
```
suggest going back to original statement for other countries.

robin-wang commented on 8_child_illness.do:
```diff
@@ -45,6 +47,24 @@ if ~inlist(name,"Philippines2017","Ethiopia2016","Haiti2016") {
+                 local lab: variable label `var' 
+        replace `var' = . if ///
+                 regexm("`lab'","(other|shop|pharmacy|market|kiosk|relative|friend|church|medical treatment|drug|addo|hilot|traditional|cs private medical|cs public sector|private mobile clinic|no treatment)") ///
+                 & !regexm("`lab'","(ngo|hospital|medical center|traditional practioner$|sub health center|health center|aid post|trained vhv and other government|maternity home|diagnostic center|wome('s|n's) consultation|(pol|po)yclinic|fap|emergency services|ambulatory/family doctor office|public mobile clinic )")  
```
Check final report.

robin-wang commented on 11_child_other.do:
```diff
@@ -2,14 +2,23 @@
 	replace c_ITN=. if ml0==.                  //Children under 5 in country where malaria is endemic (only in countries with endemic)
+		  
+	   
+	*m10 variable in Liberia2019birth label "wanted pregnancy when became pregnant",change the c_ITN according to v460(label:   children under 5 slept under mosquito bed net last night-household questionnaire)as below
```
Please review: whether we have something similar, but for TREATED bednet.

robin-wang replied:
Resolved, ml0 is still the right one to use for Liberia2019

robin-wang commented on 16_woman_cancer.do:
```diff
@@ -3,6 +3,8 @@
 ******************************   
 
 
+*note: no corresponding variables found in dataset Liberia2019
```
Propose to stick to original, and keep the template simple.

Stellaaali committed:
Update 11_child_other.do

recall the controversial change about variable ITN

Stellaaali committed:
after 1st review_check

- child_illness.do: adjustment based on report p248 p229, remain the changes
- child_other.do: delete the redundant syntax
-quality check: keep the original path

Stellaaali committed:
Merge branch 'Zhuohang-Li' of https://github.com/wengxyu1030/DHS-Recode-VII into Zhuohang-Li

robin-wang reviewed (approved):
Reviewed and approved changes.

robin-wang committed:
Merge branch 'master' into Zhuohang-Li
